<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planCustomTrip</class>
 <widget class="QDialog" name="planCustomTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: #ccc;}

QDialog QPushButton:hover
{
background: #eee;
cursor: pointer;
  
}
QDialog
{
background: #333;
}
QDialog QPushButton {  border: 2px solid #666;border-radius: 5px;z-index:1000;}

 
          
QDialog #visiting,#label_2,QSpinBox
{
 	color: #fff; 
   background: #666;
} 
#columnView
{z-index: 0; 
background:#666;
}  
#frame,QListView
{
  background:#999; 
color: #fff;
}</string>
  </property>
  <widget class="QLabel" name="visiting">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Please select Wineries to visit </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> Total miles to travel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Plan Trip</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="noWineriesSelected">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="noMilesToTravel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
